components/t547/display.py

```diff
@@ -11,7 +11,7 @@
 DEPENDENCIES = ["esp32"]
 
 CONF_GREYSCALE = "greyscale"
-
+CONF_WHITE_IS_ON = "white_is_on"
```

Comment:
I'm very open for suggestions for the configuration option name; white_is_on sounds very technical; maybe "invert_colors" or similar?